scripts/generate-vendor.sh

```diff
@@ -227,6 +227,7 @@ extract_blobs() {
       openTag=$(grep '^<?xml version' "$outPath" || true )
       if [[ "$openTag" != "" ]]; then
         grep -v '^<?xml version' "$outPath" > "$TMP_WORK_DIR/xml_fixup.tmp" || true
+        rm -f "$outPath"
```

Comment:
I don't think this is needed. The following line ensures that the contents of the "$outPath" file is always overwritten.

Reply:
`echo "$openTag" > "$outPath"` fails if the existing file is mode 0444.

Reply:
I see. Then maybe it would be a good idea to revert perms after write. Ok, will follow-up with a different CL.